### ISSUE TITLE
Getting this error:

### ISSUE THREAD

**Issue by 4lt3r3go** [Kosinkadink/ComfyUI-VideoHelperSuite#19]
```
Error occurred when executing VHS_LoadImages:

Sizes of tensors must match except in dimension 0. Expected size 512 but got size 768 for tensor number 4 in the list.
```

**Comment by 4lt3r3go** [comment]
it was working this morning, i just loaded same workflow as always and now getting this error.
havent changed anything, touched anything or installed antying.

**Comment by shreya-dama-Crafty-Apes-VFX** [comment]
@4lt3r3go were you able to resolve it? or are you still facing this issue?

**Comment by Kosinkadink** [comment]
I believe this was an issue with some images potentially containing alpha channels, and the code at the time not handling this properly when the lengths were inconsistent. It should not longer be an issue in the current version of the code, so I'll close this issue. If it still happens to you, feel free to reopen the issue.

**Comment by Musigreg** [comment]
Hi! 
I'm having the exact same issue right now.
I loaded one folder, fine. Two folders, fine... Third one : Exact same error ! And now, all folders give me this error. Even the first two ones. 
I didn't restart, I didn't add anything, I just changed the folder name and it stopped working...

Not sure what happened 

![image](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/assets/69403961/0bf033aa-7f1d-422c-9369-42246cb42f5e)
